integrations/src/test/resources/io/jenkins/plugins/casc/SchemaTest.yml

```diff
@@ -0,0 +1,6 @@
+jenkins:
+  globalNodeProperties:
```

Comment:
This seems lonely where is the matching test?

Reply:
Got rid of it :)